docs/cli-preview3/tools/dotnet-migrate.md

```diff
@@ -0,0 +1,96 @@
+---
+title: dotnet-migrate command | .NET Core SDK
```

Comment:
I noticed that "migrate" isn't listed in "dotnet --help". Agreed that this is an odd place to log that issue.